nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/queue/GetAzureQueueStorage.java

```diff
@@ -94,7 +95,7 @@
             .build();
 
     private static final List<PropertyDescriptor> properties = Collections.unmodifiableList(Arrays.asList(
-            AzureStorageUtils.STORAGE_CREDENTIALS_SERVICE, AzureStorageUtils.ACCOUNT_NAME, AzureStorageUtils.ACCOUNT_KEY, AzureStorageUtils.PROP_SAS_TOKEN,
+            AzureStorageUtils.STORAGE_CREDENTIALS_SERVICE, AzureStorageUtils.ACCOUNT_NAME, AzureStorageUtils.ACCOUNT_KEY, AzureStorageUtils.PROP_SAS_TOKEN, AzureStorageUtils.ENDPOINT_SUFFIX,
```

Comment:
@sjyang18 If you aren't using the storage credentials service the endpoint suffix doesn't have the intended effect. I think you need to make updates to the [`createCloudQueueClient()`](https://github.com/apache/nifi/blob/aa7c5e21782cf4d2f4b10ae63851c53f7c47a80f/nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/queue/AbstractAzureQueueStorage.java#L68-L74) like the changes made for `createCloudBlobClient()`.

Reply:
Thanks for catching this bug. I have fixed it.